### PR TITLE
Milestone2 - Add and remove items.

### PR DESCRIPTION
**Pull request by omarsalem7**
## What does this PR do?

This PR is raised to add and remove items from to-dos

## Description of the task to be completed

- [x] Add to-do functionality
- [x] Add remove to-do functionality
- [x] Add update to-do functionality

## How should it be tested

Clone the repository
```
git clone https://github.com/omarsalem7/Todo-list-webpack.git
```
Navigate to the folder
```
cd Todo-list-webpack
```
Checkout the branch
```
git checkout milestone2-todo
```

### CODE REVIEW

File: src/todos.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  removeTodo(todoID) {
+    this.list = this.list.filter((todo) => todo.index !== todoID);
+    this.list.forEach((todo, index) => {
+      todo.index = index;
+    });
+    console.log(this.list);
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
```
Comment (DeliceLydia) [inline src/todos.js]: - [x] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. currently when I remove one task see what happen
if I have these tasks
todo

td

When I remove `cleaning` that means `hiking` will update to the index `1` not `0`
hike

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @omarsalem7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Good README
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/todos.js (hunk 1 above)
Comment (omarsalem7) [reply src/todos.js]: Hi Lydia 🌷
Thank you for reviewing and supporting, I changed what you stated, you can check it now.
Have a nice day.
Omar 😄

**Review by aliabbani — APPROVED** [review]
Hi @omarsalem7 ✋, 

Your project is complete! 🥇 There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**